### PR TITLE
Rename InstanceExternalIp to ExternalIp

### PR DESCRIPTION
**Pull request by smklein**
Fixes https://github.com/oxidecomputer/omicron/issues/1614
Depends on https://github.com/oxidecomputer/omicron/pull/1611

### CODE REVIEW

**Review by david-crespo — APPROVED** [review]
can't argue with that

File: nexus/src/db/queries/external_ip.rs [hunk 1]
```diff
@@ -31,10 +31,9 @@ type FromClause<T> =
 type IpPoolRangeFromClause = FromClause<schema::ip_pool_range::table>;
 const IP_POOL_RANGE_FROM_CLAUSE: IpPoolRangeFromClause =
     IpPoolRangeFromClause::new();
-type InstanceExternalIpFromClause =
-    FromClause<schema::instance_external_ip::table>;
-const INSTANCE_EXTERNAL_IP_FROM_CLAUSE: InstanceExternalIpFromClause =
-    InstanceExternalIpFromClause::new();
+type ExternalIpFromClause = FromClause<schema::external_ip::table>;
+const INSTANCE_EXTERNAL_IP_FROM_CLAUSE: ExternalIpFromClause =
```
Comment (bnaecker) [inline nexus/src/db/queries/external_ip.rs:35]: Chop the `INSTANCE_` prefix from the const?

**Review by bnaecker — APPROVED** [review]
Missed one const name, but otherwise LGTM!

File: nexus/src/db/queries/external_ip.rs (hunk 1 above)
Comment (smklein) [reply nexus/src/db/queries/external_ip.rs]: Done!

**Pull request merged**